src/profile-logic/symbolication.js

```diff
@@ -562,10 +562,13 @@ export function applySymbolicationStep(
 
   // Now it is time to look at funcs.
   // For funcs belonging to a native library, we group frames into funcs based
-  // on the function name string.
+  // on the function name string and the file name.
```

Comment:
Looks like we are not using the line information to group them. I guess it's impossible to have two functions with the same name in a file, right? It would be good to comment this as well maybe?

Reply:
> Looks like we are not using the line information to group them.

No, because the symbolication API does not tell us at which line a function starts. The line number in the symbolication result is a line *inside* the function; it's the line inside the function that's currently executed. So we can't group by functionname+line - we'd get a different function for lots of frames inside the same function which are sampled at different lines.

> I guess it's impossible to have two functions with the same name in a file, right?

Yeah I think it's impossible. Or if it is possible, it falls under "if you do this, it's your own fault if the profiler treats the functions as the same."

> It would be good to comment this as well maybe?

Sure, I'll add a comment.